app/src/components/header-buttons/cancelButton.tsx

```diff
@@ -1,31 +1,35 @@
 import styled from "styled-components";
 
 const Button = styled.button`
-  display: flex;
-  align-items: center;
-  min-height: 50px;
-  padding-top: 9px;
-  padding-left: 10px;
-  background-color: transparent;
-
-  &:hover {
-      background-color: transparent;
-      border-color: transparent;
-    }
-
-  :focus {
-    outline: none;
+display: flex;
+align-items: center;
+padding-top: 9px;
+padding-left: 10px;
+background-color: transparent;
+height: 64px;
+
+&:hover {
+    background-color: transparent;
+    border-color: transparent;
   }
+
+:focus {
+  outline: none;
+}
```

Comment:
not sure I'm seeing these states when testing locally, were you able to verify?

Reply:
the focus? I am able to see no outline, yes

Reply:
Gotchya, thanks for verifying!